services/sidecar/src/simcore_service_sidecar/mpi_lock.py

```diff
@@ -104,19 +107,24 @@ def try_to_acquire_lock_factory():
         coroutine_factory=try_to_acquire_lock_factory,
     )
 
-    if managed_to_acquire_lock:
-        Thread(
-            target=thread_worker,
-            args=(
-                lock_manager,
-                lock,
-                asyncio.get_event_loop(),
-            ),
-            daemon=True,
-        ).start()
+    reply_queue.put(managed_to_acquire_lock)
+    if not managed_to_acquire_lock:
+        logger.info("No locking extention is required, skipping")
```

Comment:
same here, why not ``reply_queue.put(False)``

Reply:
same as above